.changeset/tame-bees-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rspress/theme-default": patch
+---
+
+use new Lark icon
```

Comment:
Generally we should add `chore/fix/feat: ` at the beginning.

Reply:
Done. I've been expecting a mechanism to auto-sync changeset and commit messages for a long time. 🤡